src/app.css

```diff
@@ -4,6 +4,84 @@ code {
     Menlo,
     Monaco,
     Consolas,
-    'Courier New',
+    "Courier New",
     monospace;
 }
+
+/* Style The Dropdown Button */
```

Comment:
_note:  I couldn't find an open PR on the backend to review.  For your next review request, please open one.  It should contain all the content and point from dev to main.  Therefore, I'll put my backend review comments here:_

# Backend review

- [ ] On your readme, kindly, add the required `rails db:create, rails db:migrate, rails db:seed` db commands that your users will need to set up and run the database.  That way, your users will run your app smoothly. 👍 

- [ ] `database.yml` was not included in the repo, because it has been added to `.gitignore`.  This makes it harder for your users to run the project (I had to create my own).  Since this project requires reviews and it's meant for educational purposes, it's safe to include it, please do so. If you want to use `.env` files, include instructions on your readme about how to correctly setup `database.yml`. Kindly remember that the final evaluator (or any potential employer) will need to be able to run your project without any hassle.

- [ ] OPTIONAL: Adding this for your info and convenience:

`rails db:seed` finishes with warnings:

![image](https://user-images.githubusercontent.com/57421823/208132541-69549efb-86e8-45f7-a508-ad79fa74a3bf.png)

- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

- [ ] On your readme, kindly, add detailed instructions about how to run the tests for your users.  

Warning: If I run just "rspec" I get this:

![image](https://user-images.githubusercontent.com/57421823/208133083-029bc645-cff3-4443-badb-b407409874ec.png)

### Tips and extra help

This is because the majority or your tests are swagger tests.

Solution:

```
rake rswag
```

Result:

![image](https://user-images.githubusercontent.com/57421823/208133487-9eaab02d-aad5-45e3-b3ee-a9b4b55ea008.png)

That way, your users will get no failed tests. 👍